validation/scenario_generator.ipynb

```diff
@@ -41,18 +39,82 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "DATA_URL = \"https://raw.githubusercontent.com/OxCGRT/covid-policy-tracker/master/data/OxCGRT_latest.csv\"\n",
-    "latest_df = pd.read_csv(DATA_URL,\n",
-    "                        parse_dates=['Date'],\n",
-    "                        encoding=\"ISO-8859-1\",\n",
-    "                        error_bad_lines=False)"
+    "DATA_FILE = \"tests/fixtures/OxCGRT_latest.csv\"\n",
+    "latest_df = get_raw_data(DATA_FILE, latest=True)"
```

Comment:
Loading the raw data requires a bit of processing. Moved the logic to `scenario_generator`'s `get_raw_data`